packages/cleanup-plugin/index.ts

```diff
@@ -0,0 +1,86 @@
+import {EditorPlugin, ui, data} from '@wonderlandengine/editor-api';
+import {existsSync} from 'node:fs';
+
+/**
+ * Plugin to cleanup resources with broken links.
+ */
+export default class CleanupPlugin extends EditorPlugin {
+
+    result: Record<string, Array<string>> = {};
```

Comment:
Personally, I like to use `Map<string, string[]>` instead of records. The API is a bit different, but to use it as a map it's actually faster.

Also, we generally use `string[]`, so array literal instead of the `Array` type

Reply:
Feel like this could be better typed as well. The `data.ts` file in `editor-api` could expose an interface called `Resources` and you could use those keys as an input